src/frontend/tsx/GamePanel.tsx

```diff
@@ -93,19 +93,25 @@ export function GamePanel () {
         </div>
         {/* buttons */}
         {/* offer draw button */}
-        <button
-          className='option'
-          disabled={!hasGame || displayGame.sentDrawOffer}
-          onClick={offerDrawOnClick}>
-          Offer Draw</button>
+        {!showingArchive ? (
+          <button
+            className='option'
+            disabled={!hasGame || displayGame.sentDrawOffer}
+            onClick={offerDrawOnClick}>
+            Offer Draw</button>
+        ) : (null)
+        }
         {/* resign button */}
-        <button
-          className='option'
-          disabled={!hasGame}
-          onClick={resignOnClick}>
-          Resign</button>
+        {!showingArchive ? (
+          <button
+            className='option'
+            disabled={!hasGame}
+            onClick={resignOnClick}>
+           Resign</button>
+        ) : (null)
+        }
         {/* claim special draw */}
-        {hasGame ? (
+        {!showingArchive ? (
```

Comment:
@bonbud-macryg is this alright?
I swapped `hasGame` boolean with `showingArchive`. 
This is to remove the draw and resign buttons when an archived game is being displayed. 
I couldn't think of a scenario where `hasGame` was false and Claim Special Draw would be singled out as the only button to be removed. 
Let me know if I was wrong.

Reply:
I think `hasGame` means in English "is `displayGame` non-null right now?". Until now we've been using "is `displayGame` non-null?" as a proxy for "are we viewing a game?" But with this PR maybe we should just change all instances of `hasGame` to something like `hasActiveGame`. If I'm not missing something this should achieve the same results but make the intended behaviour clearer. What do you think @rovmug-ticfyn?

Reply:
@bonbud-macryg 
I think `hasGame` as a whole is fine to stay as is. It's clear enough code.
Really I just was double checking for this particular instance where I replaced `hasGame` with my own boolean to hide draw and resign buttons when an archive game is displayed.

Reply:
Ah, in that case I think this is fine.